ios/Gemfile.lock

```diff
@@ -240,7 +240,7 @@ GEM
     terminal-table (1.8.0)
       unicode-display_width (~> 1.1, >= 1.1.1)
     thread_safe (0.3.6)
-    trailblazer-option (0.1.1)
+    trailblazer-option (0.1.0)
```

Comment:
Why did you downgrade?

Reply:
Error message due to wrong version. This way it works fine.